Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqTable" {
+
+    It "should list tables from a dataset object from pipeline."{
+        $tables = Get-GcbqDataset “pshell_testing” | Get-GcbqTable
```

Comment:
I agree with Quoc, a TODO: is a great way to show that. You can even stub out the BeforeAll and AfterAll and put the TODO in there.